packages/react-instantsearch-hooks-router-nextjs/src/index.ts

```diff
@@ -1,9 +1,15 @@
 import history from 'instantsearch.js/es/lib/routers/history';
+import { warn } from 'react-instantsearch-hooks';
 
 import type { Router, UiState } from 'instantsearch.js';
 import type { BrowserHistoryArgs } from 'instantsearch.js/es/lib/routers/history';
 import type { Router as NextRouter, SingletonRouter } from 'next/router';
 
+warn(
+  Date.now() < new Date('2023-08-09').getTime(),
+  'The package `react-instantsearch-hooks-router-nextjs` is replaced by `react-instantsearch-router-nextjs`. The API is the same, but the package name has changed. Please update your dependencies.'
```

Comment:
We could also add a link to the upgrade guide which will document codemods and other stuff

Reply:
https://www.algolia.com/doc/guides/building-search-ui/upgrade-guides/react/ would be the link then, or a specific heading?

Reply:
I think this is good enough personally